pkg/apis/dashboard/v0alpha1/types.go

```diff
@@ -66,6 +67,65 @@ type VersionsQueryOptions struct {
 	Version int64 `json:"version,omitempty"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type LibraryPanel struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// The dashboard body (unstructured for now)
+	Spec LibraryPanelSpec `json:"spec"`
```

Comment:
Question: Not sure what you mean by unstructured here, as it seems to have a structure with specific fields?

Reply:
I'll update the comment! it was unstructured at first... then I got more ambitious